src/components/title-search-filters.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Accordion, FilterAccordionHeader } from '@folio/stripes-components/lib/Accordion';
+import RadioButton from '@folio/stripes-components/lib/RadioButton';
+
+const pubtypeFilters = [
+  { label: 'All', value: 'all' },
```

Comment:
This list of filterable types will probably be better off coming from the back end, but hardcoding it in the front-end is probably an acceptable first pass.